### PR TITLE
[CAM-9752] Allow agent can connect to CAM by PROXY

### PR DESCRIPTION
**Pull request by mirasierra**
I have updated all deployment policy to insert proxy information.

@gsanchezu

### CODE REVIEW

File: Cloud Application Manager/Deploying Anywhere/using-azure.md [hunk 1]
```diff
@@ -10,12 +10,28 @@
 
 **In this article:**
 
-* [Overview](#overview)
-* [Before You Begin](#before-you-begin)
-* [Benefits of Deploying to Azure Using Cloud Application Manager](#benefits-of-deploying-to-azure-using-cloud-application-manager)
-* [Registering Your Azure Subscription in Cloud Application Manager](#registering-your-azure-subscription-in-cloud-application-manager)
-* [Deploying in Azure](#deploying-in-azure)
-* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+- [Using Azure](#using-azure)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-azure.md]: Please, leave the ToC as it was

File: Cloud Application Manager/Deploying Anywhere/using-azure.md [hunk 2]
```diff
@@ -144,10 +162,21 @@ In case of multi-tier applications deployed to different roles, follow these tip
 
 ##### Network
 
+![Resource devices section of deployment policy](../../images/cloud-application-manager/deployment-policy/azure-classic-network.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-azure.md]: Update this alternate text and all that you copied and pasted, ;)

File: Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md [hunk 3]
```diff
@@ -9,14 +9,29 @@
...
 
-* [Overview](#overview)
-* [Before You Begin](#before-you-begin)
-* [Registering Your Microsoft Azure Subscription (ARM) in Cloud Application Manager](#registering-your-microsoft-azure-subscription-arm-in-cloud-application-manager)
-* [Deploying Instances in Azure](#deploying-instances-in-azure)
-* [Registering Existing Instances from your Azure Account](#registering-existing-instances-from-your-azure-account)
-* [Azure Native Resources](#azure-native-resources)
-* [Shutdown and Terminate Instances in Azure](#shutdown-and-terminate-instances-in-azure)
-* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+- [Overview](#overview)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md]: Leave ToC unchanged

File: Cloud Application Manager/Deploying Anywhere/using-private-cloud-on-vmware-cloud-foundation.md [hunk 4]
```diff
@@ -8,13 +8,18 @@
...
 
 **In this article:**
-* [Overview](#overview)
-* [Audience](#audience)
-* [Prerequisites](#prerequisites)
-* [Register CenturyLink Private Cloud on VMware Cloud Foundation as Provider](#register-centurylink-private-cloud-on-vmware-cloud-foundation-as-provider)
-* [Deploy to CenturyLink Private Cloud from Cloud Application Manager](#deploy-to-centurylink-private-cloud-from-cloud-application-manager)
-* [Naming vCloud resources](#naming-vcloud-resources)
-* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+- [Overview](#overview)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-private-cloud-on-vmware-cloud-foundation.md]: Same as above

File: Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md [hunk 5]
```diff
@@ -9,18 +9,39 @@
...
-* [Connect your AWS Account in Cloud Application Manager](#connect-your-aws-account-in-cloud-application-manager)
-* [Create a custom AWS Policy](#create-a-custom-aws-policy)
-* [Create an IAM Role with the Policy chosen](#create-an-iam-role-with-the-policy-chosen)
-* [Add Custom AMIs in Cloud Application Manager](#add-custom-amis-in-cloud-application-manager)
-* [Deploy to your AWS Account](#deploy-to-your-aws-account)
-* [EC2 (Linux and Windows)](#ec2-linux-and-windows)
-* [AWS ECS](#aws-ecs)
-* [Shutdown and Terminate Instances in AWS](#shutdown-and-terminate-instances-in-aws)
-* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+- [Overview](#overview)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md]: Again

**Comment by mirasierra** [comment]
I have made all changes that you asked me.

File: Cloud Application Manager/Deploying Anywhere/using-azure.md [hunk 6]
```diff
@@ -10,6 +10,7 @@
 
 **In this article:**
 
+* [Using Azure](#using-azure)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-azure.md]: Please, remove this item

File: Cloud Application Manager/Deploying Anywhere/using-azure.md [hunk 7]
```diff
@@ -132,6 +133,8 @@ In case of multi-tier applications deployed to different roles, follow these tip
 
 ##### Resource
 
+![Resource devices section of deployment policy](../../images/cloud-application-manager/deployment-policy/azure-classic-resource.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-azure.md]:
```suggestion
![Resource section of deployment policy](../../images/cloud-application-manager/deployment-policy/azure-classic-resource.png)
```

File: Cloud Application Manager/Deploying Anywhere/using-azure.md [hunk 8]
```diff
@@ -144,10 +147,21 @@ In case of multi-tier applications deployed to different roles, follow these tip
 
 ##### Network
 
+![Network devices section of deployment policy](../../images/cloud-application-manager/deployment-policy/azure-classic-network.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-azure.md]:
```suggestion
![Network section of deployment policy](../../images/cloud-application-manager/deployment-policy/azure-classic-network.png)
```

File: Cloud Application Manager/Deploying Anywhere/using-azure.md [hunk 9]
```diff
@@ -144,10 +147,21 @@ In case of multi-tier applications deployed to different roles, follow these tip
...
+
 | Option | Description |
 |--------|-------------|
 | Location | Select a virtual network, region, or affinity group to physically place the instance. <li>**Virtual Networks**. To let the instance connect to services in Azure, select a virtual private network that you previously created in your subscription. See the Azure help on [creating virtual networks](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-create-vnet-classic-portal). </li><li>**Regions**. Select a region to place the instance. </li><li>**Affinity Groups**. Select an affinity group, if you created one to keep the instance close to other regional resources in the datacenter.</li> |
 
+##### Proxy
+
+![Proxy devices section of deployment policy](../../images/cloud-application-manager/deployment-policy/azure-classic-proxy.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-azure.md]:
```suggestion
![Proxy section of deployment policy](../../images/cloud-application-manager/deployment-policy/azure-classic-proxy.png)
```
Please, update for all alt texts

**Comment by mirasierra** [comment]
I have made all changes that you asked me.

File: Cloud Application Manager/Deploying Anywhere/using-cloudstack.md [hunk 10]
```diff
@@ -94,6 +94,8 @@ When you’re ready to [launch an instance](deploying-managing-instances.md) in
 
 **Resource**
 
+![Resource devices section of deployment policy](../../images/cloud-application-manager/deployment-policy/cloudstack-resource.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-cloudstack.md]: Please, remove `devices` from all these alt texts

File: Cloud Application Manager/Deploying Anywhere/using-cloudstack.md [hunk 11]
```diff
@@ -105,11 +107,24 @@ When you’re ready to [launch an instance](deploying-managing-instances.md) in
...
+
 | Option | Description |
 |--------|-------------|
 | Network | Default guest network configuration or a pre-configured network to allow traffic to the virtual machine. Consists of the permitted CIDR block of IP addresses from where traffic to the virtual machine is allowed. For more information, see the [CloudStack help](http://docs.cloudstack.apache.org/projects/cloudstack-administration/en/4.3/networking_and_traffic.html). |
 | Security Groups |	Filter incoming and outgoing traffic for the virtual machine based on a set of rules. Multiple security groups in a zone can be selected for a virtual machine. For more information, see [Security Groups](http://docs.cloudstack.apache.org/projects/cloudstack-administration/en/4.3/networking_and_traffic.html#security-groups). |
 
+
+**Proxy**
+
+![Resource devices section of deployment policy](../../images/cloud-application-manager/deployment-policy/cloudstack-proxy.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-cloudstack.md]: Update alt text

**Comment by jim-phillips** [comment]
```
File '/home/travis/build/CenturyLinkCloud/PublicKB/images/cloud-application-manager/google-deployment-manager/proxy.png' not found. Is there a file extension? Are the file extension and path correct? (Referenced from '/home/travis/build/CenturyLinkCloud/PublicKB/Cloud Application Manager/Deploying Anywhere/using-google-cloud.md')
```
Missing or badly named image file. @mirasierra

**Comment by mirasierra** [comment]
I had already uploaded the proxy.png.

File: Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md [hunk 12]
```diff
@@ -403,6 +405,8 @@ To deploy workloads to an EC2 instance, create a [deployment policy](../Automati
 
 #### Placement
 
+![Resource section of deployment policy](../../images/cloud-application-manager/deployment-policy/aws-placement.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md]:
```suggestion
![Placement section of deployment policy](../../images/cloud-application-manager/deployment-policy/aws-placement.png)
```

File: Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md [hunk 13]
```diff
@@ -411,14 +415,28 @@ To deploy workloads to an EC2 instance, create a [deployment policy](../Automati
 
 #### Network
 
+![Resource section of deployment policy](../../images/cloud-application-manager/deployment-policy/aws-network.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md]:
```suggestion
![Network section of deployment policy](../../images/cloud-application-manager/deployment-policy/aws-network.png)
```

File: Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md [hunk 14]
```diff
@@ -411,14 +415,28 @@ To deploy workloads to an EC2 instance, create a [deployment policy](../Automati
...
 | Elastic IP |	When launching to AWS, select Elastic IP to allocate a fresh static IP address from the EC2 or VPC pool and associate it to the instance depending on whether you’re deploying to EC2 classic or your VPC. If you’re using dynamic DNS to assign an IP address in EC2 or want to allow internet traffic to communicate with your instance in a non default VPC, then use Elastic IPs to guarantee public access. **Note:** You can’t autoscale the instance when you choose an Elastic IP for it. For more information, see the [AWS help](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html). |
 
+#### Proxy
+
+![Resource section of deployment policy](../../images/cloud-application-manager/deployment-policy/aws-proxy.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md]:
```suggestion
![Proxy section of deployment policy](../../images/cloud-application-manager/deployment-policy/aws-proxy.png)
```

File: Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md [hunk 15]
```diff
@@ -411,14 +415,28 @@ To deploy workloads to an EC2 instance, create a [deployment policy](../Automati
...
+
+| Option | Description |
+|-------------------|-------------|
+| Host |	The hostname or domain of the proxy that the agent will use to connect back to Cloud Application Manager, once it has been installed in the deployed instance. |
+| Port |  The port of the proxy that the agent will use to connect back to Cloud Application Manager, once it has been installed in the deployed instance. |
+
+
 #### Other
 
+![Resource section of deployment policy](../../images/cloud-application-manager/deployment-policy/aws-others.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md]:
```suggestion
![Other section of deployment policy](../../images/cloud-application-manager/deployment-policy/aws-others.png)
```

**Comment by mirasierra** [comment]
I have made all changes ask me.

**Pull request merged**